### PR TITLE
Libgdx update 

### PR DESCRIPTION
**Pull request by obusher**
* build.gradle path update to 1.5.5
* Declaration of native dependencies
* Delete of old local libraries (.so and .jar)
* Resolving update conflicts
* Introducing ./jniLibs mechanics.
* New Viewports for stretch and maximize screenmode
* Stage will not dispose actors.

Deprecated PR: #1174

### CODE REVIEW

**Comment by aried3r** [comment]
1.5.5 is the latest version.

**Comment by obusher** [comment]
[New Testrun (yellow)](https://jenkins.catrob.at/view/All-Categories/view/Catroid-multi-job/job/Catroid-Multi-Job-Custom-Branch-RELOADED/978/) for 1.5.5.

**Comment by avihric** [comment]
ok to test

File: catroid/src/org/catrobat/catroid/content/Look.java [hunk 1]
```diff
@@ -142,7 +142,7 @@ public void createBrightnessContrastShader() {
 	}
 
 	@Override
-	public void draw(SpriteBatch batch, float parentAlpha) {
+	public void draw (Batch batch, float parentAlpha) {
```
Comment (aried3r) [inline catroid/src/org/catrobat/catroid/content/Look.java]: Whitespace

**Comment by obusher** [comment]
ok to test

**Comment by obusher** [comment]
ok to test

**Comment by obusher** [comment]
[Test Green] (https://jenkins.catrob.at/view/All-Categories/view/Catroid-multi-job/job/Catroid-Multi-Job-Custom-Branch-RELOADED/1305/) 
1x Warning already fixed.

**Comment by obusher** [comment]
ok to test

**Comment by obusher** [comment]
ok to test

**Comment by obusher** [comment]
ok to test

**Pull request merged**